WooCommerce/Classes/Universal Links/Routes/OrderDetailsRoute.swift

```diff
@@ -1,11 +1,26 @@
 import Foundation
+import Yosemite
 
 /// Shows order details from a given universal link that matches the right path
 ///
 struct OrderDetailsRoute: Route {
     let path = "/orders/details"
 
     func perform(with parameters: [String: String]) {
-        DDLogInfo("We received an order details universal link with parameters: \(parameters)")
+        guard let storeIdString = parameters[ParametersKeys.blogId],
+              let storeId = Int64(storeIdString),
+              let orderIdString = parameters[ParametersKeys.orderId],
+              let orderId = Int64(orderIdString) else {
+            return DDLogError("Error: we receive an universal link for order details but parameters couldn't be parsed.")
+        }
```

Comment:
If we fail to parse parameters, I think we should continue with the original link intent which would be to open the browser with it, what do you think?

Reply:
I think it is a great idea, added in aa9291c3ce54533ea6aa54841704586f97a95c5a